docs/docs/docs_es/introduction/motivation.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Motivación
+---
+
+Por razones de compatibilidad y simplicidad, es mejor usar las funciones nativas de React para el manejo de estados, en lugar de utilizar estados externos y globales. Pero React tiene algunas limitantes:
+
+- El estado de un componente solo puede ser compartido, si éste estado se mueve hacia el componente padre que se tiene en común, pero este implica tener un árbol de componentes mas grande que renderizar.
+- El contexto de React solo puede mantener un valor, y no un conjunto indefinido de valores con sus respectivos consumidores.
+- Por estas dos razones, es difícil dividir el código de la parte más alta del árbol de componentes (donde tenemos el estado), de las hojas del mismo árbol (donde el estado es usado).
+
+Queremos mejorar esto y al mismo tiempo seguir con la misma API, semántica y comportamiento de React como sea posible.
+
+Recoil define un grafo ortogonal directo pero también intrínseco a tu árbol de componentes de React. Los cambios de estado fluyen desde las raíces de este grafo (que llamamos átomos), pasan a través de funciones puras (que hemos llamado selectores) y finalmente llegan hasta los componentes.
```

Comment:
The direct translation doesn’t make sense here: mantiene paralelamente un grafo, pero también... add comma before last y